client/client.go

```diff
@@ -393,6 +393,16 @@ func (c *Client) ExecuteRequestRaw(ctx context.Context, inputs RequestInput) (*h
 	// We will only return a response from the API it is in the HTTP StatusCode
 	// 2xx range
 	// StatusMultipleChoices is StatusCode 300
+	if resp.StatusCode >= http.StatusOK &&
+		resp.StatusCode < http.StatusMultipleChoices {
+		return resp, nil
+	}
+	// Handle Deleted Instances,  This wil return a 410 triton-go will consider this behavior not an error and return nil.
```

Comment:
Perhaps: `// GetMachine returns a HTTP 410 response for deleted instances, but the body of the response is still a valid machine object with a State value of "deleted".  Return the object to the caller instead of an error.`

Reply:
Replaced